### DISCUSSION TITLE
How to debug nodejs fetch request?

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [trpc/trpc#4881] [Q&A]
How to debug nodejs fetch request?

**Comment by Aysenur-Erkin** [comment]
When making a fetch request, imagine that we want to see how our code is executing step by step and the information related to the request. Here's where we can use console.log(). When we initiate the request, we want to see where the request is going and which URL it's targeting. We do this using console.log(). Then as we receive the response, we want to check important details like the status of the response and the data that is returned. So we use console.log() while inspecting the response. Finally in case there's any error, we add an error handling (catch) block to see the error message. This way we can trace every step of our code and quickly identify any potential issues related to fetch requests.
